Updating licenses, access, and RBAC (#6310)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-cloud-apis/service-tokens.md

### Team plans using service account tokens

> The following permissions can be assigned to a service account token on a Team plan. See the [Enterprise permissions](/docs/cloud/manage-access/enterprise-permissions) article for more information about what these roles are able to do. 

Reviewer: Suggest using "refer to" and simplifying the sentence.
```suggestion
The following permissions can be assigned to a service account token on a Team plan. Refer to [Enterprise permissions](/docs/cloud/manage-access/enterprise-permissions) for more information about these roles. 
```

- Account Admin &mdash; Account Admin service tokens have full `read + write` access to an account, so please use them with caution.  A Team plan refers to this permission set as an "Owner role."
- Job Admin 
- Job Runner
- Metadata Only
- Member
- Read-only 
- Semantic Layer Only 

### Enterprise plans using service account tokens

> The following permissions can be assigned to a service account token on an Enterprise plan. For more details about these permissions, see "[Enterprise permissions](/docs/cloud/manage-access/enterprise-permissions)."

Reviewer: Suggest making this consistent with above wording
```suggestion
Refer to [Enterprise permissions](/docs/cloud/manage-access/enterprise-permissions) for more information about these roles. 
```
